distributed/deploy/spec.py

```diff
@@ -99,6 +99,54 @@ async def finished(self):
     def __repr__(self):
         return f"<{type(self).__name__}: status={self.status}>"
 
+    def _repr_html_(self, heading=None):
+        if self.status == Status.created:
+            status = "Created"
+            bg_color = "#caf0f8"
+            border_color = "#48cae4"
+        elif self.status == Status.running:
+            status = "Running"
+            bg_color = "#c7f9cc"
+            border_color = "#78c6a3"
+        elif self.status == Status.closed:
+            status = "Closed"
+            bg_color = "#ffbfad"
+            border_color = "#ff6132"
+
+        heading = heading or self.__class__.__name__
```

Comment:
Given that the class name will likely be `Worker` or `Scheduler` for basically every implementation we probably want to include the full class path. You can use `dask.utils.typename` for this.

```python
dask.utils.typename(type(self))
```

Reply:
Oh wow! Didn't know about this. This makes it easier :+1: 

Added it.